tensorflow/tools/xla_extract/tf_graph_to_xla_lib.cc

```diff
@@ -177,14 +188,15 @@ xla::HloModuleProto ExtractHloFromGraphDef(const GraphDef& in_graph,
 
   for (int l = 0; l < fdef_ground_truth.size(); l++) {
     if (new_xla_args[l].name == "") {
-      LOG(FATAL) << "name mismatch error for " << fdef_ground_truth[l].name();
+      LOG(ERROR) << "name mismatch error for " << fdef_ground_truth[l].name();
     }
   }
 
   xla_args = new_xla_args;
   // we no longer need to do the rotation
-
-  LOG(INFO) << "xla args in correct order and matches fdef\n";
+  if(xla_log >= 2){
```

Comment:
what's the significant of "2"?  It appears a lot, so maybe deserving of a #define or constexpr with a name?

Reply:
oh this is just a way to keep track of logging for now.  ive commented it in the code, what 0,1,2 do.

Reply:
if it were a named constant, it'd be self-commenting in line :)

Reply:
haha true :)